src/__tests__/index.test.ts

```diff
@@ -1,233 +1,100 @@
-import { afterEach, beforeEach, describe, expect, jest, test } from '@jest/globals';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+
+import { strictEqual } from 'node:assert';
+import { after, before, beforeEach, describe, Mock, mock, test } from 'node:test';
 import { Collection, Db } from 'mongodb';
-import Papr from '../index';
-import * as model from '../model';
 import { schema } from '../schema';
 import types from '../types';
-import { VALIDATION_ACTIONS, VALIDATION_LEVEL } from '../utils';
+import { expectToBeCalledOnceWith } from './assert';
 
 describe('index', () => {
-  let db: Db;
+  let db: Omit<Db, 'collection' | 'collections' | 'command' | 'createCollection'> & {
```

Comment:
worth making an extra `Mock` utility type that does this for us? i.e. takes an object and re-types its properties as mocked versions?